tests/CoaddPsf.py

```diff
@@ -534,6 +510,48 @@ def testValidPolygonPsf(self):
             self.assertTrue(testRelDiff(m1, m1coadd, 0.01))
             self.assertTrue(testRelDiff(m2, m2coadd, 0.01))
 
+    def testGoodPix(self):
+        """Demonstrate that we can goodPix information in the CoaddPsf"""
+        cd11, cd12, cd21, cd22  = 5.55555555e-05, 0.0, 0.0, 5.55555555e-05
+        crval1, crval2 = 0.0, 0.0
+        bboxSize = afwGeom.Extent2I(2000, 2000)
+        crpix = afwGeom.PointD(bboxSize/2.0)
+        crval = afwCoord.Coord(afwGeom.Point2D(crval1, crval2))
+        wcsref = afwImage.makeWcs(crval, crpix, cd11, cd12, cd21, cd22)
+
+        schema = afwTable.ExposureTable.makeMinimalSchema()
+        schema.addField("weight", type="D", doc="Coadd weight")
+        schema.addField("goodpix", type="I", doc="Number of good pixels")
+        mycatalog = afwTable.ExposureCatalog(schema)
+
+        # Create several records, each with its own peculiar center and numGoodPixels
+        xwsum = 0
+        ywsum = 0
+        wsum = 0
+        for i, (xOff, yOff, numGoodPix) in enumerate((
+            (30.0, -20.0, 25),
+            (32.0, -21.0, 10),
+            (28.0, -19.0, 30),
+        )):
+            xwsum -= xOff * numGoodPix
+            ywsum -= yOff * numGoodPix
+            wsum += numGoodPix
+            record = mycatalog.getTable().makeRecord()
+            record.setPsf(measAlg.DoubleGaussianPsf(25, 25, 10, 1.00, 0.0))
```

Comment:
When I see numbers hard-coded like this in a test case, I often find it helpful if there's an accompanying comment explaining why those particular numbers were chosen -- are they completely arbitrary placeholders, or do they have some significance regarding the test conditions? In this case, it's pretty clear what's happening so I don't really object, but maybe it's worth considering.